src/main.py

```diff
@@ -251,19 +226,19 @@ def _run_testing(env: BaseEnv, approach: BaseApproach) -> Metrics:
             continue
         num_found_policy += 1
         try:
+            if CFG.make_videos:
```

Comment:
Great change! Let's change this in `tests/envs/test_cover.py` too, inside `run_policy_with_simulator_on_task`

Reply:
oh, you were way ahead of me haha